src/iop/colorchecker.c

```diff
@@ -1352,11 +1342,7 @@ void gui_init(struct dt_iop_module_t *self)
     dt_bauhaus_combobox_add(g->combobox_patch, cboxentry);
   }
 
-  GtkWidget *hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, DT_BAUHAUS_SPACE);
-  gtk_box_pack_start(GTK_BOX(hbox), g->combobox_patch, TRUE, TRUE, 0);
-
-  g->colorpicker = dt_color_picker_new(self, DT_COLOR_PICKER_POINT_AREA, hbox);
-  g_signal_connect(G_OBJECT(g->colorpicker), "toggled", G_CALLBACK(_color_picker_callback), self);
+  dt_color_picker_new(self, DT_COLOR_PICKER_POINT_AREA, g->combobox_patch);
```

Comment:
It might be useful to add
`  gtk_widget_set_name(g->combobox_patch, "keep-active");
`
that causes the color picker to not be deactivated when using any of the controls (for example switching from relative to absolute)